core/store/kv/default_test.go

```diff
@@ -11,8 +10,10 @@ import (
 	"golang.org/x/xerrors"
 )
 
+const delaTestDir = "dela-core-kv"
+
 func TestBoltDB_UpdateAndView(t *testing.T) {
-	dir, err := ioutil.TempDir(os.TempDir(), "dela-core-kv")
+	dir, err := os.MkdirTemp(os.TempDir(), delaTestDir)
```

Comment:
it would have been nice to make this kind of changes in a separate PR 😅

Reply:
Yes, I would've definitely left the update to Go 1.19 to another PR 🙃.